recipes/vaex-ml/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "vaex-ml" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "ae1f2c50c4883e42c99922351f2903ff7ad4078085c606ce7411b0696f88c1f2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
```

Comment:
Do you really need a build section? If so, better add a comment.

Reply:
No, i guess no, thanks!